lib/cc/analyzer/container.rb

```diff
@@ -124,6 +124,15 @@ def container_data(duration: nil, status: nil)
           @stderr_io.string
         )
       end
+
+      def reap_running_container
+        container_id = `docker ps --quiet --filter name=#{@name}`.chomp
```

Comment:
Doubtful. The behavior I'm referring to has been in Docker for a long time now. Example:

```
$ docker run -d --name=foo alpine sleep 120
219b77e8f943b41c4ca61461c123af62f2e7533a5ac23fa23734c75c739e67e1

☺ ~ 
 [osx 18:41:36] $ docker kill foo
foo
```

(That's off 1.8.2)